tests/UtilitiesTest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Tests;
+
+use App\Utilities;
+use PHPUnit\Framework\TestCase;
+
+class UtilitiesTest extends TestCase
+{
+    public function testParsePrettyTimestamp()
```

Comment:
I think each of these cases should be their own tests so we know which cases do and don't work should that method be updated in the future.

Reply:
I'm sorry for the delayed response. Notifications aren't working how I want them to. PHPUnit will give a proper error for each test case that goes wrong so to divide them into separate methods seems to be a bit of an overkill.

Reply:
If the first assertion were to fail, would the second and nth assertions run? 

If the answer is fail immediately, then I would argue each assertion should be their own test.

Testing is ultimately about the reporting. What do you want your report to look like?

I could make a branch and work up an example of what I think this should look like if you're interested.

Reply:
No it doesn't fail immediately, I agree that would be very inconvenient.